content/reference/user/reports/index.md

```diff
@@ -58,13 +58,20 @@ Each tab has the same structure except the summary that is available only for re
 
 This bar is a combination of buttons:
 
-- **Start / Abort**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)
-- **Grafana**: Link to the Grafana dashboard if you have filled in the configuration in frontline.conf
+- **Start / Stop**: Use this button to start a new run of the simulation, or stop the ongoing run (not available if you have a Viewer permission)
 - **Generate public link**: To create a public link
+- **Export**: To [export]({{< ref "../reports#export" >}}) reports
+
+{{< img src="run-bar.png" alt="Run bar" >}}
+
+### Sub Run Bar
```

Comment:
This is not a run bar, as we don't run simulation from here, maybe tab bar?

Reply:
The purpose of those buttons is to filter the data present in the chart, maybe the `filter bar`?

Reply:
yes, that's a better name